Week1/Week1/ViewController.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  ViewController.swift
+//  Week1
+//
+//  Created by 정종인 on 2022/04/12.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+
+
+    lazy var textField1: UITextField = {
+        let textField = UITextField()
+        
+        textField.borderStyle = .roundedRect
+        textField.placeholder = "숫자1 입력"
+        
+        return textField
+    }()
+    
+    lazy var textField2: UITextField = {
+        let textField = UITextField()
+        
+        textField.borderStyle = .roundedRect
+        textField.placeholder = "숫자2 입력"
+        
+        return textField
+    }()
+    
+    lazy var performPlusButton: UIButton = {
+        let button = UIButton()
+        
+        button.backgroundColor = .systemBlue
+        button.setTitle("더하기 수행", for: .normal)
+        button.titleLabel?.textColor = .white
+        button.tag = Tag.plus
+        
+        return button
+    }()
+    
+    lazy var performMinusButton: UIButton = {
+        let button = UIButton()
+        
+        button.backgroundColor = .systemCyan
+        button.setTitle("빼기 수행", for: .normal)
+        button.titleLabel?.textColor = .white
+        button.tag = Tag.minus
+        
+        return button
+    }()
+    
+    lazy var performMultipleButton: UIButton = {
+        let button = UIButton()
+        
+        button.backgroundColor = .systemBrown
+        button.setTitle("곱하기 수행", for: .normal)
+        button.titleLabel?.textColor = .white
+        button.tag = Tag.multiple
+        
+        return button
+    }()
+    
+    lazy var performDivideButton: UIButton = {
+        let button = UIButton()
+        
+        button.backgroundColor = .systemIndigo
+        button.setTitle("나누기 수행", for: .normal)
+        button.titleLabel?.textColor = .white
+        button.tag = Tag.divide
+        
+        return button
+    }()
+    
+    lazy var resultLabel: UILabel = {
+        let label = UILabel()
+        
+        label.textAlignment = .center
+        
+        return label
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        makeViews()
+        makeLogics()
+    }
+    
+    private func makeViews() {
+        self.view.addSubview(textField1)
+        textField1.snp.makeConstraints {
+            $0.top.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.large)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
```

Comment:
```suggestion
      $0.leading.trailing.equalTo(self.view.safeAreaLayoutGuide).inset(Size.Margin.small)
```
이렇게 하는게 더 깔끔할 것 같네요!

Reply:
inset은 처음 알았네요! 감사합니다~